Runtime/Shaders/MapsBaker.cs

```diff
@@ -60,13 +60,31 @@ internal static class MapsBaker {
 
         // If there is no such texture, create one with the given fallback value:
 
-        float value = fallbackPropertyValue;
-        if (materialProperties.TryGetValue(fallbackPropertyNameID, out IMaterialPropertyData fallbackMaterialProperty))
-            value = fallbackMaterialProperty.floatValue;
+        Color value = new Color(
+            fallbackPropertyValue,
+            fallbackPropertyValue,
+            fallbackPropertyValue,
+            fallbackPropertyValue);
```

Comment:
Does transparent color make sense as a fallback value ?
Perhaps the alpha should be made to 1 as a fallback ?

Reply:
These all need to be the same unless they come from a colour value. The fallback is defined in a property we receive from the DCC tool, not something hardcoded in this case.